src/tests/unit/tests/injected/analyzer-controller.test.ts

```diff
@@ -243,10 +243,10 @@ describe('AnalyzerControllerTests', () => {
 });
 
 class AnalyzerStub implements IAnalyzer<any> {
-    public analyze() {
+    public analyze(): Error {
         throw new Error('Method not implemented.');
     }
-    public teardown() {
+    public teardown(): Error {
```

Comment:
shouldn't this be `void`?

Reply:
It doesn't matter for this case, where `throw new Error()` is the only command in the function and I am fine with this function returning `Error`. Until you feel very strongly about me returning `void` here, I guess we can keep this as `Error`

Reply:
changed it to "void"